plugins/offboard/include/plugins/offboard/offboard.h

```diff
@@ -105,6 +105,17 @@ class Offboard : public PluginBase {
                                  clock-wise looking from above). */
     };
 
+    /**
+     * @brief Type for attitude commands in NED coordinates (roll, pitch, yaw and thrust
```

Comment:
I think it's in `body coordinates`, so it's relative to the drone frame not the absolute frame, right?

```suggestion
     * @brief Type for attitude commands in body coordinates (roll, pitch, yaw and thrust).
```

Reply:
This part is a bit confusing. I made a a new commit with. rewording.

According to PX4/Firmware/msg/vehicle_attitude_setpoint.msg

```
float32 roll_body		# body angle in NED frame (can be NaN for FW)
float32 pitch_body		# body angle in NED frame (can be NaN for FW)
float32 yaw_body		# body angle in NED frame (can be NaN for FW)
```

New description:
*@brief Type for attitude body angles in NED reference frame (roll, pitch, yaw and thrust)*